src/coreclr/jit/lsra.h

```diff
@@ -2063,6 +2100,17 @@ class LinearScan : public LinearScanInterface
     {
         return this->rbmMskCalleeTrash;
     }
+    //------------------------------------------------------------------------
+    // getLowGprRegs(): Return the set of GPR registers associated with non APX
+    // encoding only, i.e., remove the eGPR registers from the available
+    // set.
+    //
+    // Return Value:
+    // Register mask of non APX GPR registers.
+    FORCEINLINE SingleTypeRegSet getLowGprRegs() const
```

Comment:
I don't think you need this method do you? All the accesses are within `LinearScan` class, so can just use the field `lowGprRegs` directly.

Reply:
Done